src/main/resources/export/cpp/vscode_launch.json

```diff
@@ -0,0 +1,21 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
JSON doesn't allow for comments in the RFC spec. Is this legal in this case?

Reply:
Also fixed.  These files were generated by VS Code, so they will work. However, I have cleaned them up, and retested the results.